.jfrog.taskcat.yml

```diff
@@ -3,14 +3,14 @@ project:
   owner: quickstart-eng@amazon.com
   s3_object_acl: private
   s3_regional_buckets: True
-  s3_bucket: tcat-34c17b83dcbd547393a86d8053a8d298 # commercial accounts (also for GovCloud)
-  #s3_bucket: tcat-a3e80b6745b2547da1c745b16adf2a66    # aws-seller accounts (MarketPlace)
+  s3_bucket: tcat-qs-ec2 # commercial accounts (also for GovCloud)
+  #s3_bucket: tcat-qs-ec2    #aws-seller accounts (MarketPlace)
 
   parameters:
-    KeyPairName: "giri"
+    #KeyPairName: "" #Key Pair Name which should be already exists in aws
```

Comment:
Should mention the use of Taskcat override file here in the comment.

Reply:
Sure